templates/zerver/help/import-from-slack.md

```diff
@@ -269,8 +271,9 @@ If you imported your organization into Zulip Cloud, simply e-mail
 [support@zulip.com](mailto:support@zulip.com) to request this.
 
 !!! warn ""
-    To avoid confusion, first make sure that
-    the users in your organization are aware that their Slack account has been
+
+    To avoid confusion, first make sure that the users in your
+    organization are aware that their workspace account has been
```

Comment:
Let's just make this "their account" -- "their workspace account" sounds a bit off to me.